sickbeard/helpers.py

```diff
@@ -1409,7 +1409,11 @@ def get_size(start_path='.'):
     for dirpath, dirnames, filenames in ek.ek(os.walk, start_path):
         for f in filenames:
             fp = ek.ek(os.path.join, dirpath, f)
-            total_size += ek.ek(os.path.getsize, fp)
+            try:
+                total_size += ek.ek(os.path.getsize, fp)
+            except OSError, e:
+                logger.log(u"Unable to get size for file " + fp + ": " + repr(e) + " / " + str(e),
```

Comment:
Here I would recommend using 2 log line:
1. Error log line to inform the user of the error:
  `logger.log('Unable to get size for file {filePath}. Error msg is: {errorMsg}'.format(filePath=fp, errorMsg=str(e)), logger.ERROR)`
2. A debug line for the traceback(for us to be able to debug where is was thrown):
   `logger.log(traceback.format_exc(), logger.DEBUG)`

Don't forget to import traceback if it isn't